.golangci.yml

```diff
@@ -172,6 +172,9 @@ linters-settings:
       - name: bool-literal-in-expr
       - name: constant-logical-expr
 
+  ginkgolinter:
+    forbid-focus-container: true
+
```

Comment:
Note: this is for https://github.com/nunnatsa/ginkgolinter#focus-container--focus-individual-spec-found-bug

Removing the config here won't make any change for us currently. 

Either we remove it here or propose to also add it to CAPI :-) @sbueringer wdyt?

Reply:
I'm not sure. The purpose of this rule is to prevent leaking of focus containers (from local debug environment) to the codebase.

We can add it to capi as well. The reason it's not th hrre is that this rule is new to v1.54.

Reply:
ack, I just learned about this config option too.

One target of #2058 is to make the golangci lint config as similar as possible to the one used by CAPI so we don't have to maintain two different versions :-)

Reply:
OK. Opened https://github.com/kubernetes-sigs/cluster-api/pull/9320 (:

Reply:
Sounds good, thank you!